R/frequency_encoder.R

```diff
@@ -0,0 +1,26 @@
+#' frequency_encoder
+#'
+#' This function encodes categorical variables using the frequencies of each category.
+
+#'
+#' @param X_train A tibble representing the training data set containing some categorical features/columns.
+#' @param X_test A tibble representing the test set, containing some set of categorical features/columns.
+#' @param y A numeric vector or character vector representing the target variable. If the objective is "binary", then the vector
```

Comment:
The frequency encoder does utilize anything about y in this case, just the distributions of the categories. Can remove y as a parameter.